README.md

```diff
@@ -0,0 +1 @@
+### Repository for playing with the triagebot
```

Comment:
sounds like some content is missing in this README

Reply:
@rustbot author

Reply:
@rustbot ready

Reply:
@rustbot author

Reply:
@rustbot ready

Reply:
@rustbot author

Reply:
@rustbot author

Reply:
@rustbot ready

Reply:
well well well

@rustbot ready

Reply:
> well well well well
> 
> @rustbot ready

@rustbot ready